.eslintignore

```diff
@@ -1,3 +1,6 @@
 **/*.d.ts
-tests
+coverage
+dist
 lib
+node_modules
+tests
```

Comment:
You're sure you don't want to lint the tests?

Reply:
There is no tests for now but good point, I will remove it